sam-test-project/template.yaml

```diff
@@ -0,0 +1,60 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: >-
+  sam-test-project
+
+# Transform section specifies one or more macros that AWS CloudFormation uses to process your template
+# https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/transform-section-structure.html
+Transform: ["AWS::Serverless-2016-10-31", "CfMacroSlicWatch"]
```

Comment:
DX comment: is there any way that we can replace `CfMacroSlicWatch` with simply `SlicWatch`?

I think `CfMacro` is a bit redundant and it just makes it less readable (and harder to type if not copy-pasting)

Reply:
Replaced to 'SlicWatch'